packages/platforms/browser/lib/config.ts

```diff
@@ -79,6 +80,11 @@ export function createSchema (hostname: string, defaultRoutingProvider: RoutingP
       defaultValue: defaultSendPageAttributes,
       message: 'should be an object',
       validate: isSendPageAttributes
+    },
+    tracePropagationUrls: {
```

Comment:
I feel like the JS SDK is a candidate for configuring through the callback, as is mentioned in the PD:

> We will decide on a SDK-by-SDK basis whether the config setting should be at the SDK level or at a network-request-instrumentation level.

Given that developers will be controlling whether to generate a span for a network request using the callback and `requestInfo` return value, does it feel odd to control the presence of the trace header by regex rather than a new field in `requestInfo` (e.g. `requestInfo.includeTraceHeader`?

Reply:
I agree with this. I think `propagateTraceContext` would be the right term.